<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_2</class>
 <widget class="QDialog" name="Window_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computer Vision</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon.png</normaloff>Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 248, 250);
color: rgb(20, 23, 26);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="toolTip">
    <string>Go Back</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#backButton
{
color: #fff;
border-radius: 10%;
font-size: 24px;
font-weight: bold;
background-color: rgb(29, 161,242);
}
#backButton:hover
{
background-color: rgb(50, 119, 194);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/arrow-left.svg</normaloff>Icons/arrow-left.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="eye_detection">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>65</y>
     <width>210</width>
     <height>330</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>210</width>
     <height>330</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#eye_detection
{
border-radius: 100%;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Photos/eye_detection.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="color_detection">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>65</y>
     <width>210</width>
     <height>330</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>210</width>
     <height>330</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Photos/color_detection.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="object_detection">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>65</y>
     <width>210</width>
     <height>330</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>210</width>
     <height>330</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Photos/object_detection.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="eye_text">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>25</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#eye_text
{
font-family: 'Roboto', sans-serif;
font-style: italic;
font-size: 24px;
}</string>
   </property>
   <property name="text">
    <string>Eye Detection</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="color_text">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>25</y>
     <width>210</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>1</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#color_text
{
font-family: 'Roboto', sans-serif;
font-style: italic;
font-size: 24px;
}</string>
   </property>
   <property name="text">
    <string>Color Detection</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="object_text">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>25</y>
     <width>210</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#object_text
{
font-family: 'Roboto', sans-serif;
font-style: italic;
font-size: 24px;
}</string>
   </property>
   <property name="text">
    <string>Object Detection</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="go_eye_Button">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>410</y>
     <width>128</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#go_eye_Button
{
color: #fff;
border-radius: 20%;
font-family: 'Roboto', sans-serif;
font-style: italic;
font-size: 24px;
background-color: rgb(29, 161,242);
}
#go_eye_Button:hover
{
background-color: rgb(50, 119, 194);
}</string>
   </property>
   <property name="text">
    <string>GO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_color_Button">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>410</y>
     <width>128</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#go_color_Button
{
color: #fff;
border-radius: 20%;
font-family: 'Roboto', sans-serif;
font-style: italic;
font-size: 24px;
background-color: rgb(29, 161,242);
}
#go_color_Button:hover
{
background-color: rgb(50, 119, 194);
}</string>
   </property>
   <property name="text">
    <string>GO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_object_Button">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>410</y>
     <width>128</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#go_object_Button
{
color: #fff;
border-radius: 20%;
font-family: 'Roboto', sans-serif;
font-style: italic;
font-size: 24px;
background-color: rgb(29, 161,242);
}
#go_object_Button:hover
{
background-color: rgb(50, 119, 194);
}</string>
   </property>
   <property name="text">
    <string>GO</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
